app/src/main/java/org/oppia/app/utility/FontScaleConfigurationUtil.kt

```diff
@@ -0,0 +1,33 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import android.content.Context.WINDOW_SERVICE
+import android.util.DisplayMetrics
+import android.view.WindowManager
+import org.oppia.app.model.StoryTextSize
+
+object FontScaleConfigurationUtil {
```

Comment:
Can we make this an injectable class instead of a static class? That will make it easier for us to switch its implementation, such as for tests, if needed.

Reply:
Done.